spec/requests/google_analytics/presence_spec.rb

```diff
@@ -6,13 +6,14 @@
 
   let(:ga_identifiers) do
     [
-      "<script async src='https://www.google-analytics.com/analytics.js'></script>",
+      "<script src=\"https://www.google-analytics.com/analytics.js\" nonce=\"noncevalue\" async=\"async\"></script>",
```

Comment:
I know you hate `%(string)` style strings but they have their use! 😂

Reply:
Don't know why I didn't use single quotes in the first place :sweat_smile: 

There, I just saved my self hundreds of hours from googling what `%` literal I should use for a plain string. :joy: